chore(issue template): attempting to fix two entries/links (#25881)
apache/superset: .github/ISSUE_TEMPLATE/sip.md

---
name: SIP

> about: "Superset Improvement Proposal. See SIP-0 (https://github.com/apache/superset/issues/5602) for details. The purpose of a Superset Improvement Proposal (SIP) is to introduce any major change into Apache Superset, such as a major new feature, subsystem, or piece of functionality, or any change that impacts the public interfaces of the project"

Author: ```suggestion
about: "Superset Improvement Proposal. See SIP-0 (https://github.com/apache/superset/issues/5602) for details. A SIP introduces any major change into Apache Superset's code or process."
```

Author: ![image](https://github.com/apache/superset/assets/812905/b14b7adc-d405-4c7d-87a1-ecc1ea2844dd)

The description was too long.... I've shortened it :)
